test/run-k8s-integration.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Can you add a version of this script to run against a local project?

Reply:
done